danilo_abbasciano.py

```diff
@@ -0,0 +1,20 @@
+from microbit import *
+
+SECRET_CODE='ABBA'
+
+display.show(Image.CONFUSED)
+
+INPUT_CODE=''
+
+while (len(INPUT_CODE) < 4):
```

Comment:
the problem is: when you just press the button, the read string is AAAAAAAAA immediately, so there is no time to read the B button. Maybe with a delay?